README.md

```diff
@@ -2,24 +2,250 @@
 
 # PMIx Standard
 
-This repository contains the LaTeX source for the PMIx standard document, and discussions regarding the PMIx standard.
+PMIx is an application programming interface (API) specification to provide
+High Performance Computing (HPC) libraries and programming models with
+portable and well defined access to commonly available
+distributed computing system services.
 
-## Contributing to the PMIx Standard
+## Governance
 
-There are two main ways to contribute to the PMIx Standard: Participating in the discussion, and writing sections of the document.
+The PMIx Standard is governed by an Administrative Steering Committee (ASC) comprised of representatives from a consortium of research, academic, and industrial organizations.The ASC shall consist of one representative from each Member organization, as defined below, and shall perform the following functions:
 
-### Participating in the discussion
+* Define the mission of the PMIx Standard.
+* Plan release timelines for versions of the PMIx Standard based on available resources, target features, and target goals of the Member organizations.
```

Comment:
It is unclear what the target goals of the member organizations have to do with release time lines. As an example, a feature shouldn't be rushed just because a particular member organization has promised it to customers by a certain time -- the release cycles of the standard should be independent of the business of the implementations. It is also unclear on what 'available resources' means. Do you mean that release time lines could be altered, say slowed if there weren't enough people to make enough progress to warrant a release?

Reply:
The intent here is to capture a little reality in terms of how the ASC works. There is no point in defining a target release date if there aren't people available to review things and move them along. Witness the lengthy delays in getting other standards out.

Similarly, the reality is that releases of the standard are always going to be driven by the target goals and features of the Members. They are the ones voting, and they are primarily the ones proposing new features. As a result, the release timeline is largely going to be driven by their business needs, not by arbitrary dates.

So while in principle things should be separate, the reality is that things tend to be rather coupled. Nothing different from what we see in other standard bodies. This bullet just states that one function of the ASC is to propose and manage release timelines that reflect the needs of its members.